src/View/Home/index.html.twig

```diff
@@ -1,9 +1,12 @@
 {% extends 'layout.html.twig' %}
 
 {% block stylesheet %}
-    <link rel="stylesheet" type="text/css" href="/assets/css/index.css"/> 
```

Comment:
use the twig parent functionality in order to keep the header / footer stlesheet inside the stylesheet content